Pcap++/Makefile

```diff
@@ -63,18 +63,21 @@ endif
 ifdef LIBPCAP_INLCUDE_DIR
 INCLUDES += -I"$(LIBPCAP_INLCUDE_DIR)"
 endif
+ifdef BUILD_WITH_STATIC_LIBPCAP
+INCLUDES += -I"$(STATIC_LIBPCAP_INCLUDE)"
+endif
 
 ifdef HAS_PCAP_IMMEDIATE_MODE
 DEPS += -DHAS_PCAP_IMMEDIATE_MODE
 endif
 
 ifdef USE_DPDK
-FLAGS := -msse -msse2 -msse3 -mssse3
+CXXFLAGS += -msse -msse2 -msse3 -mssse3

```

Comment:
I'm not sure why you changed `FLAGS` to `CXXFLAGS`

Reply:
`FLAGS` was only used in the call to the compiler below. I replaced it with `CXXFLAGS` but now it doesn't make sense anymore. Now it should be `PCAPPP_BUILD_FLAGS`. Fixing it.